lib/plugins/aws/deploy/compile/events/apiGateway/lib/methods.js

```diff
@@ -351,7 +383,7 @@ module.exports = {
                 "RequestParameters" : {},
                 "Integration" : {
                   "IntegrationHttpMethod" : "POST",
-                  "Type" : "AWS",
+                  "Type" : "${integrationType}",
```

Comment:
Good one 👍  (acutally I was thinking the same when implementing this).

Turns out that the whole JSON String is wrapped in back-ticks (if you look a few lines above).